### PR TITLE
fix farsi trans

### PR DESCRIPTION
**Pull request by Sosha1996**
fixing some farsi word

### CODE REVIEW

**Comment by mapsmetest** [comment]
Can one of the admins verify this patch?

**Comment by mpimenov** [comment]
Please sign the CLA: https://maps.me/cla/

File: data/categories.txt [hunk 1]
```diff
@@ -8556,6 +8556,7 @@ uk:^Платіжний термінал
 vi:^Thiết bị thanh toán
 zh-Hans:^支付终端
 zh-Hant:^付款終端
+fa:دستگاه کارخوان
```
Comment (mpimenov) [inline data/categories.txt]: Are you sure it's
دستگاه کارخوان
and not
دستگاه کارتخوان
?

File: data/categories.txt [hunk 2]
```diff
@@ -8621,6 +8622,7 @@ uk:^Пункт постачання води
 vi:^Điểm cấp nước
 zh-Hans:^取水点
 zh-Hant:^飲水站
+fa:شیر آب
```
Comment (mpimenov) [inline data/categories.txt]: https://wiki.openstreetmap.org/wiki/Tag:amenity=water_point
Not that it's "drinking water" not just any "water".
Is it true for what you suggest?

File: data/categories.txt [hunk 3]
```diff
@@ -8917,6 +8919,7 @@ uk:^Ритуальна служба
 vi:^Tổ chức tang lễ
 zh-Hans:^殡仪馆
 zh-Hant:^葬儀社
+fa:مسئول تشیع جنازه
```
Comment (mpimenov) [inline data/categories.txt]: Are you sure it's
مسئول تشیع جنازه
and not
مسئول تشییع جنازه
?

**Comment by mpimenov** [comment]
@Sosha1996 Ping. Sorry for the delay.

**Comment by mpimenov** [comment]
@Sosha1996 Please update your Pull Request

**Comment by mpimenov** [comment]
@Sosha1996 ping

**Comment by mpimenov** [comment]
@Sosha1996 Please rebase the PR onto the master branch

**Comment by mpimenov** [comment]
@Sosha1996 Thank you. In order to merge the request we had to rebase it. See https://github.com/mapsme/omim/pull/10795

**Pull request closed without merging**